ios/FluentUI/Bottom Sheet/BottomCommandingController.swift

```diff
@@ -0,0 +1,596 @@
+            bottomSheetHeroStackTopConstraint?.constant = bottomSheetHeroStackTopMargin
+        } else {
+            moreButtonView.isHidden = !isExpandable
+        }
+    }
+
+    private lazy var moreButtonView: UIView = {
+        let moreButtonItem = TabBarItem(title: Constants.BottomBar.moreButtonTitle, image: Constants.BottomBar.moreButtonIcon ?? UIImage())
+        let moreButtonView = TabBarItemView(item: moreButtonItem, showsTitle: true)
+        moreButtonView.alwaysShowTitleBelowImage = true
+
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleMoreButtonTap(_:)))
+        moreButtonView.addGestureRecognizer(tapGesture)
+
+        NSLayoutConstraint.activate([
+            moreButtonView.widthAnchor.constraint(equalToConstant: Constants.heroButtonWidth),
+            moreButtonView.heightAnchor.constraint(equalToConstant: Constants.heroButtonHeight)
+        ])
+
+        return moreButtonView
+    }()
+
+    private lazy var heroCommandStack: UIStackView = {
+        let itemViews = heroItems.map { createAndBindHeroCommandView(with: $0) }
+        let stackView = UIStackView(arrangedSubviews: itemViews)
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.addInteraction(UILargeContentViewerInteraction())
+
+        isHeroCommandStackLoaded = true
+        return stackView
+    }()
+
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .grouped)
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.separatorStyle = .none
+        tableView.alwaysBounceVertical = false
+        tableView.isAccessibilityElement = true
+        tableView.sectionFooterHeight = 0
+        tableView.backgroundColor = Constants.tableViewBackgroundColor
+        tableView.register(TableViewCell.self, forCellReuseIdentifier: TableViewCell.identifier)
+        tableView.register(TableViewHeaderFooterView.self, forHeaderFooterViewReuseIdentifier: TableViewHeaderFooterView.identifier)
+        tableView.delegate = self
+        tableView.dataSource = self
+
+        isTableViewLoaded = true
+        return tableView
+    }()
+
+    // MARK: - Command tap handling
+
+    @objc private func handleHeroCommandTap(_ sender: UITapGestureRecognizer) {
+        guard let tabBarItemView = sender.view as? TabBarItemView, let binding = viewToBindingMap[tabBarItemView] else {
+            return
+        }
+
+        switch binding.item.commandType {
+        case .toggle:
+            tabBarItemView.isSelected.toggle()
+            binding.item.isOn = tabBarItemView.isSelected
+            fallthrough
+        case .simple:
+            binding.item.action(binding.item)
+        }
+    }
+
+    @objc private func handleMoreButtonTap(_ sender: UITapGestureRecognizer) {
+        let popoverContentVC = UIViewController()
+        popoverContentVC.view.addSubview(tableView)
```

Comment:
or just popoverContentVC.view = tableView? But generally with UITableViews iOS makes things so much easier for us if we just use UITableViewController

Reply:
We only need to have it in a controller when it's hosted in a popover. In the sheet it also has the hero stack and probably a grid in the future, so we don't really need a UITableViewController which would force me to implement the datasource and delegate in there, forcing me to somehow pipe through relevant item data from this main controller.